src/actions.ts

```diff
@@ -0,0 +1,38 @@
+// TODO: This is a copy of this: https://github.com/redux-observable/redux-observable/blob/master/src/ActionsObservable.js
+// Remove after this is resolved: https://github.com/redux-observable/redux-observable/issues/95
+import { Injectable, Inject } from '@angular/core';
+import { Action, Dispatcher } from '@ngrx/store';
+import { Observable } from 'rxjs/Observable';
+import { Operator } from 'rxjs/Operator';
+import { filter } from 'rxjs/operator/filter';
+
+
+@Injectable()
+export class Actions extends Observable<Action> {
+  constructor(@Inject(Dispatcher) actionsSubject: Observable<Action>) {
+    super();
+    this.source = actionsSubject;
+  }
+
+  lift(operator) {
```

Comment:
Operators in rxjs use the `lift` method to alter the behavior of the source observable. By implementing `lift` on custom observables, the result of applying an operator to an observable will return an instance of your custom observable.

This makes it technically possible to do something like:

```ts
this.actions$.merge(something).ofType('Some Action')
```

Note that after doing `.merge()` I get back an instance of `Actions` instead of just a regular `Observable`